takwimu/takwimu_ui/src/Theme.js

```diff
@@ -47,7 +59,7 @@ const Theme = createMuiTheme({
     },
     subtitle1: {
       fontFamily: FONT_FAMILY_TEXT,
-      fontSize: '1.75rem',
+      fontSize: '1.25rem',
```

Comment:
You're not using this, why are you changing it? It's being used in **Where to next** section and it needs to be `1.75rem`